wg-security-audit/sig-security-charter-proposal.md

```diff
@@ -0,0 +1,90 @@
+- How, when and where vulnerabilities are announced
+- Defining the criteria and process for supporting Kubernetes subprojects, such as the [dashboard](https://github.com/kubernetes/dashboard), [ingress-nginx](https://github.com/kubernetes/ingress-nginx), or [kops](https://github.com/kubernetes/kops).
+
+#### Security Community Management and Outreach
+
+Provide an entry point to the Kubernetes community for new security-minded contributors, as well as a meeting point to discuss security themes and issues within Kubernetes, including:
+
+- Work with [SIG Contributor Experience](https://github.com/kubernetes/community/tree/master/sig-contributor-experience) to curate and staff security discussion channels (e.g. slack channel, mailing list, discourse, stack overflow, etc.).
+- Answer security questions from inexperienced users (that don't know what SIG to go to), and identify common questions or issues as areas for improvement.
+- Provide an "entry point" for new contributors interested in security. Route these new contributors to other SIGs when they have more specific goals (e.g. SIG Node for container isolation).
+
+#### Horizontal Security Documentation
+
+Author and maintain cross-cutting security documentation, such as hardening guides and security benchmarks. Seek out and coordinate with experts in other SIGs for input on the documentation (i.e. we go to them, they don't need to come to us). In-scope documentation includes:
+
+- Hardening guides and best practices
+- Security benchmarks
+- Improving documentation to address common misunderstandings or questions
+- Threat models
+
+#### Security Audit
+
+Manage recurring security audits and follow up on issues. Coordinate vendors to perform the audit and publish the findings. Follow up on issues with the affected SIG and help coordinate resolution, which can include:
+
+- Helping to prioritize the fixes, possibly by recruiting from SIG Security (while acknowledging that the ultimate authority in deciding whether and how to fix an issue lies with the responsible SIG).
+- Documenting mitigations, workarounds, or caveats, especially when the responsible SIG decides not to fix a reported issue.
+
+### Out of scope
+
+In contrast to SIG Auth, SIG Security does not own any Kubernetes cluster component code. 
+
+Further, SIG Security’s scope does not include:
+
+- Kubernetes authentication, authorization, audit and security policy features.  (SIG Auth)
+- Private vulnerability response (belongs to the PSC), including:
+- Embargoed vulnerability management
+- Bug bounty submission triage and management
+- Non-public vulnerability collection, triage, and disclosure
+- The mechanisms to protect confidentiality/integrity of API data (belongs to SIG API Machinery, SIG Auth or others)
+- Security audit for all other CNCF projects (e.g., etcd, CoreDNS, CRI-O, containerd)  (Belongs to the CNCF’s SIG Security.) 
+- Any projects outside of the Kubernetes project
```

Comment:
Do we consider this to be limited to https://github.com/kubernetes projects and not https://github.com/kubernetes-sigs?

Reply:
What if we changed line 64 to say:
`- Any process or community support for projects outside of https://github.com/kubernetes and https://github.com/kubernetes-sigs`

Would that be more clear?

Reply:
@micahhausler ^^^

Reply:
Kubernetes also encompasses other github orgs (ref: https://github.com/kubernetes/community/tree/master/github-management#actively-used-github-organizations)

I would suggest referencing that list and explicitly listing any exclusions here

Reply:
Thank you, @spiffxp ! Will do now.

Reply:
I'm good with `github.com/kubernetes` and `github.com/kubernetes-sigs` for starters. We can add/remove from that at later times if desired

Reply:
What if line 64 reads?

- Any process or community support for projects outside of https://github.com/kubernetes, https://github.com/kubernetes-client, https://github.com/kubernetes-csi, and https://github.com/kubernetes-sigs.  Note: this list is generated by excluding  kubernetes-incubator, kubernetes-retired and the private kubernetes-security from the [Kubernetes project's Actively Used Organizations](https://github.com/kubernetes/community/tree/master/github-management#actively-used-github-organizations).

Reply:
/lgtm

Reply:
/lgtm

Reply:
OK - that's it then.